lambdas/backend-api/src/__tests__/templates/app/template-client.test.ts

```
@@ -1472,6 +1668,38 @@ describe('templateClient', () => {
       });
     });
 
+    test('should return a failure result, when proofing is not enabled', async () => {
```

this looks identical to the above `should return a failure result, when proofing is disabled` - if it's not can you make the test names clearer